extensions/wikia/AdEngine3/src/setup.js

```diff
@@ -108,6 +108,7 @@ function setupAdContext(wikiContext, isOptedIn = false, geoRequiresConsent = tru
 		isGeoEnabled('wgAdDriverUnstickHiViLeaderboardAfterTimeoutCountries')
 	);
 
+	context.set('services.confiant.enabled', isGeoEnabled('wgAdDriverConfiantDesktopCountries'));
```

Comment:
`isGeoEnabled('wgAdDriverConfiantCountries') && isGeoEnabled('wgAdDriverConfiantDesktopCountries')` to be consistent with mobile?

Reply:
No, because `wgAdDriverConfiantCountries` is a copy of mobile wgVar. By doing this we can accidentally enable it on desktop.

Reply:
Talked over offline. `wgAdDriverConfiantCountries` will be removed after rollout.